controllers/awsmachine_controller.go

```diff
@@ -186,7 +187,48 @@ func (r *AWSMachineReconciler) SetupWithManager(mgr ctrl.Manager, options contro
 			&source.Kind{Type: &infrav1.AWSCluster{}},
 			&handler.EnqueueRequestsFromMapFunc{ToRequests: handler.ToRequestsFunc(r.AWSClusterToAWSMachines)},
 		).
-		WithEventFilter(pausePredicates).
+		WithEventFilter(predicate.Funcs{
```

Comment:
These common predicates are still identical (other than logging messages), planning on refactoring them back out to helpers.go again.